From ef1d5ceefcbfacf5dfe94534c4879ca814b130f0 Mon Sep 17 00:00:00 2001
From: Jimi Cullen <jimifhcullen@gmail.com>
Date: Sun, 15 Apr 2018 15:31:31 +0000
Subject: [PATCH] Add hooks for new pycodestyle checks.

Add hooks for new checks introduced in pycodestyle 2.4.0.
For details of the changes to pycodestyle, see:
https://pycodestyle.readthedocs.io/en/latest/developer.html
---
 setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.py b/setup.py
index c4862e9..b156feb 100644
--- a/setup.py
+++ b/setup.py
@@ -119,6 +119,8 @@ setuptools.setup(
             PEP8_PLUGIN('python_3000_raise_comma'),
             PEP8_PLUGIN('python_3000_not_equal'),
             PEP8_PLUGIN('python_3000_backticks'),
+            PEP8_PLUGIN('python_3000_invalid_escape_sequence'),
+            PEP8_PLUGIN('python_3000_async_await_keywords'),
         ],
         'flake8.report': [
             'default = flake8.formatting.default:Default',
-- 
2.18.0

